website/client-old/js/controllers/inviteToGroupCtrl.js

```diff
@@ -34,9 +34,17 @@ habitrpg.controller('InviteToGroupCtrl', ['$scope', '$rootScope', 'User', 'Group
 
       if (inviteMethod === 'email') {
         var emails = _getEmails();
+        if(emails.length < 1) {
```

Comment:
I considered that. The only reason I didn't do it that way is that it would show the success message if an email invitation had a UUID, or if an invitation by UUID had only an email.